frontend/lib/laletterbuilder/homepage.tsx

```diff
@@ -18,28 +22,158 @@ export function getLaLetterBuilderImageSrc(
+      <div className="hero-body">
+        <div className="container jf-has-text-centered-tablet">
+          <h2 className="is-spaced has-text-info">
+            <Trans>Legally vetted</Trans>
+          </h2>
+          <br />
+          <p className="subtitle">
+            <Trans>
+              We built the LA Letter Builder with lawyers and non-profit tenants
+              rights organizations in Los Angeles to ensure that your letter
+              gives you the most protections.
+            </Trans>
+          </p>
+          <br />
+          <div className="is-spaced has-text-info">
+            <Trans>Created by</Trans>
+            <StaticImage
+              ratio="is-3by1"
+              src={getLaLetterBuilderImageSrc(
+                "justfix-saje-combined-logo-black"
+              )}
+              alt="JustFix SAJE"
+            />
+          </div>
+        </div>
+      </div>
+    </section>
+    <section className="jf-laletterbuilder-landing-section-tertiary">
+      <div className="hero-body">
+        <div className="container jf-has-text-centered-tablet">
+          <h2 className="is-spaced">
+            <Trans>How it works</Trans>
+          </h2>
+          <div className="text-section">
+            <p className="subtitle">
+              <Trans>Build your letter</Trans>
+            </p>
+            <label>
+              <Trans>
+                Answer some basic questions about your situation, and we will
+                automatically build a letter for you.
+              </Trans>
+            </label>
+          </div>
+          <div className="text-section">
+            <p className="subtitle">
+              <Trans>Send for free</Trans>
+            </p>
+            <label>
+              <Trans>
+                We will send the letter to your landlord or property manager for
+                free via certified mail. Or you can opt to print and mail it
+                yourself.
+              </Trans>
+            </label>
+          </div>
+          <div className="text-section">
+            <p className="subtitle">
+              <Trans>Follow up</Trans>
+            </p>
+            <label>
               <Trans>
-                This is a test localization message for LaLetterBuilder.
+                We'll check in with you via text message, see if your situation
+                improved and offer next steps.
               </Trans>
-            </small>
+            </label>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section className="jf-laletterbuilder-landing-section-primary">
+      <div className="hero-body">
+        <div className="container jf-has-text-centered-tablet">
+          <h2 className="is-spaced">
+            <Trans>Frequently asked questions</Trans>
+          </h2>
+          <Accordion
+            question={li18n._(t`What situations does the tool cover?`)}
+          >
+            Example answer here
```

Comment:
just flagging this will need to be internationalized once there is real content here

Reply:
yep, will do once i put the content in there